src/Schema/Column.php

```diff
@@ -463,4 +463,12 @@ public function toArray()
             'comment' => $this->_comment,
         ], $this->_platformOptions, $this->_customSchemaOptions);
     }
+
+    public function cloneWithName(string $name): Column
```

Comment:
This method is only needed for the deprecation layer? I think, we should mark it as `@internal` then and remove it in 4.0.